src/dagmc.cpp

```diff
@@ -609,14 +630,37 @@ void DAGUniverse::uwuw_assign_material(
   fatal_error("DAGMC was not configured with UWUW.");
 #endif // OPENMC_UWUW
 }
+
+void DAGUniverse::override_assign_material(std::string key,
+  moab::EntityHandle vol_handle, std::unique_ptr<DAGCell>& c) const
+{
+
+  // if Cell ID matches an override key, use it to override the material
+  // assignment else if UWUW is used, get the material assignment from the DAGMC
+  // metadata
+  // Notify User that an override is being applied on a DAGMCCell
+  write_message(fmt::format("Applying override for DAGMCCell {}", c->id_), 8);
+  if (material_overrides.count("id_" + std::to_string(c->id_))) {
```

Comment:
To be honest, I'm not a fan of matching based on prefixes in the material names. We should be able to provide a direct mapping of which material should be applied in place of the one provided in the DAGMC file.

Reply:
Sadly I can't set an attribute to an int.
what I did is add a "Cell_" prefix that I detect and remove when parsing the xml in the cpp"